src/types/index.ts

```diff
@@ -184,9 +182,10 @@ export type BookmarkModelInput = {
   label?: string
   cmsId?: string
 }
-/*  Bookmark refers to an existing user-created object as it is stored in MongoDB 
-    It has the addition of isRemoved, which is used to determine if the bookmark is deleted or hidden */
-export type Bookmark = {
+
+/*  MongoBookmark refers to an existing bookmark as it is stored in MongoDB. This includes
+    both user-created bookmarks and bookmarks that the user has added from the CMS.  */
+export type MongoBookmark = {
```

Comment:
I know we had briefly talked in eng sync about naming for this, MongoBookmark vs UserBookmark vs <something else>. @gidjin did you have a preference or strong opinions?

Reply:
My preference is for `UserBookmark` but I'm okay if it stays this way.

Reply:
I left it as `MongoBookmark` because, since we have bookmarks coming from two different places, it makes it more clear when working with a bookmark where it's coming from. Happy to change it if the need arises, but thought I'd clarify my intentions.

Reply:
I'm good with that reasoning. Thanks for noting it!